dev/docker/docker-compose-register.yml

```diff
@@ -50,16 +48,15 @@ services:
     profiles: ["dual"]
     env_file:
       - ../local.env
-    volumes:
-      - ../environments/anvil.json:/cfg/anvil.json
     command:
       [
-        "--config-file=/cfg/anvil.json",
+        "--config-file=config://anvil",
```

Comment:
The `config://` in the `config-file` URL is a little weird to me. Open to adding a second flag to the CLI to specify an env. Or maybe a better name? I toyed with `env://`